core/jobs.py

```diff
@@ -1411,11 +1409,6 @@ def _clear_inactive_realtime_layer(cls, latest_created_on_datetime):
     @classmethod
     def _kickoff_batch_job(cls):
         """Create and enqueue a new batch job."""
-        if job_models.JobModel.do_unfinished_jobs_exist(cls.__name__):
```

Comment:
I don't think we can start a job if another one is running. We will be getting an exception from here:
https://github.com/oppia/oppia/blob/d1d8e53bed78e7716b8af30752e7ff278bec5aa4/core/jobs.py#L1476

Reply:
I believe these two are on separate code paths: this method is for one-off/batch jobs, the other is for continuously running jobs. @seanlip is that understanding correct?

Reply:
Bus isn't this method defined under BaseContinuousComputationManager?

Reply:
Hmm seems so, I'm not sure what to make of this then. What happens if you remove the other exception, and rely on this one instead? If an exception is still raised, seems fine to remove this. Otherwise, we should figure out why this one is still around.

Reply:
OK this is strange. I did what you said -- First I did start_computation and again did start_computation. No exception was raised. I wonder how I can call these lines